src/bot_functions.py

```diff
@@ -0,0 +1,50 @@
+from pymessenger import Bot
+
+import json
+
+DEFINED_COMMANDS_PATH = "data/commands.json"
+
+BOT_COMMANDS = "commands"
+USER_TO_BOT_INPUT = "input"
+BOT_TO_USER_OUTPUT = "output"
+
+
+class MessengerBot:
+    def __init__(self, page_access_token) -> None:
+        self.bot = Bot(page_access_token)
+
+    def load_json_file(self, path: str) -> dict:
+        """Returns dictionary from .json file"""
+        with open(path) as file:
+            data = json.load(file)
+
+        return data
+
+    def answear_text_message(self, text: str) -> str:
+        """answearing for user command"""
+
+        formatted_message = text.lower().strip()
+
+        data = self.load_json_file(DEFINED_COMMANDS_PATH)
+
+        for sentence in data[BOT_COMMANDS]:
+            if formatted_message == sentence[USER_TO_BOT_INPUT]:
+                return sentence[BOT_TO_USER_OUTPUT]
+
+    def process_message(self, payload: dict) -> None:
+        """Processing user message. If user message is recognizable send answear """
+
+        event = payload["entry"][0]["messaging"]
```

Comment:
Are You sure that these keys "entry" and "messaging" are always present? Consider using "get" method instead. Also is it possible that the entry has more than one element? (0-th element)